command/agent/exec/server.go

```diff
@@ -60,6 +60,14 @@ type Server struct {
 	exitCh chan int
 }
 
+type ProcessExitError struct {
```

Comment:
```suggestion
type processExitError struct {
```
Might want to make it unexported